packages/workbox-background-sync/Queue.mjs

```diff
@@ -202,27 +285,7 @@ class Queue {
       }
       // If the browser doesn't support background sync, retry
       // every time the service worker starts up as a fallback.
-      this.replayRequests();
-    }
-  }
-
-  /**
-   * Registers a sync event with a tag unique to this instance.
-   *
-   * @private
-   */
-  async _registerSync() {
-    if ('sync' in registration) {
-      try {
-        await registration.sync.register(`${TAG_PREFIX}:${this._name}`);
-      } catch (err) {
-        // This means the registration failed for some reason, possibly due to
-        // the user disabling it.
-        if (process.env.NODE_ENV !== 'production') {
-          logger.warn(
-              `Unable to register sync event for '${this._name}'.`, err);
-        }
-      }
+      this._onSync(this);
```

Comment:
Same as previous comment re: `this.onSync({queue: this})`

Reply:
Done.